CITATION.cff

```diff
@@ -0,0 +1,21 @@
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+
+cff-version: 1.2.0
+title: Stockfish
+message: >-
+  Please cite this software using the metadata from this
+  file.
+type: software
+authors:
+  - family-names: The Stockfish developers (see AUTHORS file)
```

Comment:
```suggestion
  - name: The Stockfish developers (see AUTHORS file)
```

Reply:
I'm not sure that field can be changed, it is part of the format of the file I think. In principle, the format wants real names of all authors. That's not possible in our case, as we don't have real names of all authors, of just a github handle, and I don't feel like selecting just a handful of names does justice to the other authors. Hence this version.

Reply:
The format distinguishs between persons and entities: https://github.com/citation-file-format/citation-file-format/blob/main/schema-guide.md#authors. I think the latter is the better choice in our case because it avoids strange formatting of the citation due to the missing `given-names` field.